src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/actions/PlayActions.kt

```diff
@@ -202,12 +202,12 @@ class RefreshProjects : DumbAwareAction(AllIcons.Actions.Refresh) {
     override fun update(e: AnActionEvent) {
         super.update(e)
         val host = e.getHost()
-        e.presentation.isVisible = host?.isUnrealEngineSolution ?: false
+        e.presentation.isVisible = false
```

Comment:
I think there's "isVisibleAndEnabled" field available. Move it under if (host == null)